src/TurbulenceClosures/turbulence_closure_implementations/nothing_closure.jl

```diff
@@ -5,3 +5,6 @@
 
 calculate_diffusivities!(diffusivities, ::Nothing, args...) = nothing
 calculate_diffusivities!(::Nothing, ::Nothing, args...) = nothing
+
+@inline viscosity(::Nothing, ::Nothing) = 0.0
+@inline diffusivity(::Nothing, ::Nothing, ::Val{id}) where id = 0.0
```

Comment:
```suggestion
@inline viscosity(::Nothing, ::Nothing) = 0
@inline diffusivity(::Nothing, ::Nothing, ::Val{id}) where id = 0
```

Reply:
Change applied. Thanks for the suggestion. Can I ask why `0` is preferred here though?

Reply:
Multiplying with `0` does not impose type conversion. See, e.g.,

```Julia
julia> a = Float32(10)
10.0f0

julia> b = Float64(10)
10.0

julia> typeof(0.0*a)
Float64

julia> typeof(0.0*b)
Float64

julia> typeof(0*a)
Float32

julia> typeof(0*b)
Float64
```

Reply:
Ah, I didn't know. Thanks for the tip/info!

Reply:
It doesn't work always...

```Julia
julia> typeof(0 * Int16(3))
Int64
```

Reply:
I guess 0 is `Int64` and thus multiplying `Int64` with `Int16` imposes conversion to `Int64`. :)

Reply:
> It doesn't work always...
> 
> ```julia
> julia> typeof(0 * Int16(3))
> Int64
> ```

Hmm, interesting. Although our models are going to be float regardless so if that only happens for Ints then I guess it doesn't matter for us